keras/layers/convolutional.py

```diff
@@ -1426,16 +1432,27 @@ class ZeroPadding1D(Layer):
     def __init__(self, padding=1, **kwargs):
         super(ZeroPadding1D, self).__init__(**kwargs)
         self.padding = padding
+
+        if isinstance(padding, int):
+            self.left_pad = padding
+            self.right_pad = padding
+        elif isinstance(padding, dict):
+            self.left_pad = padding.get('left_pad', 1)
```

Comment:
Default values should be 0 (i.e. if argument isn't specified then no padding occurs on that side).